### PR TITLE
(operate) feat: add operate app

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
## Proposed changes

1) Migrated operate app to monorepo (no improvements/refactoring for the agents page as didn't have time unfortunately)
2) Created shared components: Caption (grey text with 14px font-size) and Label/TextWithTooltip (the Label on is grey - for forms, the Text one is with no styles - can be used anywhere, but currently suits for table header)
3) Moved useNominees and useNomineesMetadata to shared libs (still would like to discuss how to better organise our libs)
4) Added new table with staking contracts as per design

https://operate-cc8osymr3-autonolas.vercel.app/contracts

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: apps/operate/components/Contracts/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      !!nominees &&
+      !!metadata &&
+      !!maxNumServicesList &&
+      !!serviceIdsList &&
+      !!rewardsPerSecondList &&
+      !!minStakingDepositList &&
+      !!numAgentInstancesList
+    ) {
+      return nominees.map((item, index) => {
+        const maxSlots = Number(maxNumServicesList[index]);
```
Comment (Tanya-atatakai) [inline apps/operate/components/Contracts/hooks.ts:100]: @kupermind please feel free to only review this file, or just simply confirm if the values are correct on the screen
image

**Comment by Tanya-atatakai** [comment]
no idea why tests are failing, they work locally 🥲 will check

**Review by truemiller — APPROVED** [review]

File: apps/operate/components/Contracts/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+import { RETAINER_ADDRESS } from 'libs/util-constants/src';
+import { STAKING_TOKEN } from 'libs/util-contracts/src';
+import { getAddressFromBytes32, getBytes32FromAddress } from 'libs/util-functions/src';
+
+const ONE_YEAR = 1 * 24 * 60 * 60 * 365;
+
+const AVAILABLE_ON: Record<Address, StakingContract['availableOn']> = {
+  '0x000000000000000000000000ef44fb0842ddef59d37f85d61a1ef492bba6135d': 'pearl',
+  '0x000000000000000000000000389b46c259631acd6a69bde8b6cee218230bae8c': 'quickstart',
+  '0x0000000000000000000000005344b7dd311e5d3dddd46a4f71481bd7b05aaa3e': 'quickstart',
```
Comment (kupermind) [inline apps/operate/components/Contracts/hooks.ts:16]: Shall we have more separation in names, as now they are the same?

File: apps/operate/components/Contracts/hooks.ts (hunk 1 above)
Comment (kupermind) [reply apps/operate/components/Contracts/hooks.ts:99]: This must be multiplied by 100, and divided by the sum of (1 + number of agent instances)

**Review by kupermind — APPROVED** [review]

File: apps/operate/components/Contracts/hooks.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  return {
+    contracts,
+    isLoading:
+      isNomineesLoading ||
+      isMetadataLoading ||
+      isMaxNumServicesLoading ||
+      isServiceIdsLoading ||
+      isRewardsPerSecondLoading ||
+      isMinStakingDepositLoading ||
+      isNumAgentInstancesLoading,
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply apps/operate/components/Contracts/hooks.ts:146]: totally

**Pull request merged**